File: OLSource1/v1/VS_officedev/codesnippet/CSharp/how-to--programmatically-sort-data-in-worksheets_3.cs
            Excel.Range Fruits = Application.get_Range("A1", "B3");
                Fruits.Sort(
                Fruits.Columns[1], Excel.XlSortOrder.xlAscending,
                Fruits.Columns[2], missing, Excel.XlSortOrder.xlAscending,
                missing, Excel.XlSortOrder.xlAscending,
                Excel.XlYesNoGuess.xlNo, missing, missing,
                Excel.XlSortOrientation.xlSortColumns,
                Excel.XlSortMethod.xlPinYin,
                Excel.XlSortDataOption.xlSortNormal,
                Excel.XlSortDataOption.xlSortNormal,
                Excel.XlSortDataOption.xlSortNormal); 

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/how-to--programmatically-sort-data-in-worksheets_3.vb
        Dim Fruits As Excel.Range = Me.Application.Range("A1", "B2")
        Fruits.Sort( _
            Key1:=Fruits.Columns(1), Order1:=Excel.XlSortOrder.xlAscending, _
            Key2:=Fruits.Columns(2), Order2:=Excel.XlSortOrder.xlAscending, _
            Orientation:=Excel.XlSortOrientation.xlSortColumns, _
            Header:=Excel.XlYesNoGuess.xlNo, _
            SortMethod:=Excel.XlSortMethod.xlPinYin, _
            DataOption1:=Excel.XlSortDataOption.xlSortNormal, _
            DataOption2:=Excel.XlSortDataOption.xlSortNormal, _
            DataOption3:=Excel.XlSortDataOption.xlSortNormal)